### PR TITLE
 python3Packages.plexapi: 4.1.2 -> 4.3.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 4.3.1

Change log: https://github.com/pkkid/python-plexapi/releases/tag/4.3.1

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112343` at 9c0c3fd3 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/5a963040fd91c7605577e553d576898a">python37Packages.plexapi</a></li></li>
    <li><a href="https://gist.github.com/ed8254f9ffc5474c5207acd4172e3234">python38Packages.plexapi</a></li></li>
    <li><a href="https://gist.github.com/684d988af9f6733a42ab8b143d02ad8f">python39Packages.plexapi</a></li></li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112343` at 9c0c3fd3 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/cd5cbe074970f34a601f41b5226f4b58">python37Packages.plexapi</a></li></li>
    <li><a href="https://gist.github.com/49859ece8a916ab3d2db7f6016e3daf2">python38Packages.plexapi</a></li></li>
    <li><a href="https://gist.github.com/8184e3e148f19870bf091aa9f4c536f7">python39Packages.plexapi</a></li></li>
  </ul>
</details>

File: pkgs/development/python-modules/plexapi/default.nix [hunk 1]
```diff
@@ -1,21 +1,35 @@
-{ lib, buildPythonPackage, fetchFromGitHub, requests
-, tqdm, websocket_client, pytest, pillow, isPy27 }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+, tqdm
+, websocket_client
+, pytestCheckHook
+, pillow
```
Comment (mweinelt) [inline pkgs/development/python-modules/plexapi/default.nix:8]:
```suggestion
```

Since we don't run tests these two can be dropped.

**Review by mweinelt — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/plexapi/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/plexapi/default.nix]: Thanks, I missed to remove them after check if there are tests which could be run.

**Pull request merged**